README.md

```diff
@@ -10,19 +10,18 @@ This codeception module is based on [oqq/codeception-email-mailhog](https://gith
 brings nearly the same functionality for Mailpit as the mentioned modules did for MailHog.
 
 ## Installation
-Through composer, require the package:
+With Composer:
 ```shell
-composer req koehnlein/codeception-email-mailpit --dev
+composer require koehnlein/codeception-email-mailpit --dev
 ```
-Then turn it on in your Codeception suite yaml file
+Then enable it in your `Functional.suire.yml`:
```

Comment:
It's not necessarily required to use `Functional.suite.yml`. You could also use the module in acceptance tests and it's even possible to configure the module in the global `codeception.yml`

Reply:
OK. But I think it's better to tell people the actual file name; this is also the way the Codeception docs are referring to those files.

Reply:
I checked https://codeception.com/docs/modules/Db as example and cannot confirm, it's done this way in the Codeception docs.